trulens_eval/trulens_eval/tests/groundedness_smoke_tests.ipynb

```diff
@@ -0,0 +1,224 @@
+{
```

Comment:
<p>It would be good to show a few more models evaluated here also. Many people ask how different models perform for evals, and it's nice to be able to point to our documented smoke tests.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/truera/trulens/pull/527/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>yes will add more</p>